test-network/scripts/deployCC.sh

```diff
@@ -40,11 +40,11 @@ if [ "$CC_SRC_PATH" = "NA" ]; then
 	if [ "$CC_NAME" = "basic" ]; then
 		echo $'\e[0;32m'asset-transfer-basic$'\e[0m' chaincode
 		CC_SRC_PATH="../asset-transfer-basic"
-	elif [ "$CC_NAME" = "secure" ]; then
+	elif [ "$CC_NAME" = "secured" ]; then
```

Comment:
@nikhil550 These changes were already made in your prior commit, need to resolve the conflicts.

But you could update line 54 to also say "secured" :-)

Reply:
This file was added by accident, but I will put in the change. Moved to new PR: https://github.com/hyperledger/fabric-samples/pull/270